### ISSUE TITLE
request: a PLAN doc containing more specifics about project goals

### ISSUE THREAD

**Issue by uvtc** [MoeOrganization/moe#24]
I think it would be useful if the project had a PLAN.md file containing sections something like:
- primary general project goals with some details (AST, MOP, etc.)
- Perl 5 features/functions/isms most likely to be dropped/not-implemented
- planned incompatible syntax/semantics changes from Perl 5
- new language features/additions planned for Moe

**Comment by stevan** [comment]
Yup, this document is in the works. Mostly it is in my head right now, but will likely get into bits before I go to bed tonight.

**Comment by stevan** [comment]
For the time being I suggest referring to slide 127-137 (or 140-150, same thing though) of the talk (https://speakerdeck.com/stevan_little/perl-is-not-dead-it-is-a-dead-end)

**Comment by stevan** [comment]
John,

Moe has been moved to https://github.com/MoeOrganization/moe and you have been made a committer to the repo. This means you have push and pull rights, please use them responsibly and enjoy.

Thanks for your contribution, I look forward to many more in the future :)
- Stevan

**Comment by uvtc** [comment]
I see that there are now a number of wiki pages outlining various positions on what to jettison/keep. (Perhaps these will eventually be coalesced into a PLAN doc?)

To add my 2 cents right here, my biggest must-haves would be:
- built-in repl
- built-in way to alias long module names (ex. `use Whatchamacallit::fuse as fuse;`)
- standard simple way right from the get-go for creating, versioning, metadata-ing, building, packaging, distributing, and installing modules (including a way to install modules into somewhere in my $HOME dir rather than only into site_perl).
- strict/warnings on by default

---John

**Comment by stevan** [comment]
John, 

The idea of the wiki pages are to let everyone put in their two cents without having it be anything official. At some point in the near future we will be assembling the PLAN doc, but want to avoid anything official yet.

Ideally you could take your list and move it into the wiki do it won't be forgotten about.

Thanks,

Stevan

**Comment by uvtc** [comment]
Created my humble wiki page. Thanks. :)
